package.json

```diff
@@ -58,8 +58,11 @@
   },
   "dependencies": {
     "@circles/core": "^2.10.10",
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
     "@material-ui/core": "^4.12.4",
     "@material-ui/lab": "^4.0.0-alpha.61",
+    "@mui/material": "^5.8.4",
```

Comment:
it looks suspicious for me..
so we are adding additional mui library here? Because we still have @material-ui/core...
Here are migrations steps:
https://mui.com/material-ui/migration/migration-v4/

Reply:
I started this migration but it was too much work for now. This dual use fixes the warning though. Do you prefer we fix the warning in another way until we fully migrate mui?

Reply:
I changed to another approach :)